mir-module/src/main/resources/META-INF/resources/js/mir/mycore2orcid.js

```diff
@@ -1,73 +1,98 @@
-var orcidStatusURL  = webApplicationBaseURL + "rsc/orcid/status/";
-var orcidPublishURL = webApplicationBaseURL + "rsc/orcid/publish/";
+(function(){
+	const orcidStatusURL  = webApplicationBaseURL + "rsc/orcid/status/";
+	const orcidPublishURL = webApplicationBaseURL + "rsc/orcid/publish/";
+	const translateURL = webApplicationBaseURL + "rsc/locale/translate/";
 
-var orcidIcon = "<img alt='ORCID iD' src='" + webApplicationBaseURL + "images/orcid_icon.svg' class='orcid-icon' />";
+	const orcidIcon = "<img alt='ORCID iD' src='" + webApplicationBaseURL + "images/orcid_icon.svg' class='orcid-icon' />";
 
-var orcidTextStatusIsInProfileTrue  = "Diese Publikation ist in Ihrem ORCID-Profil vorhanden";
-var orcidTextStatusIsInProfileFalse = "Diese Publikation ist NICHT in Ihrem ORCID-Profil vorhanden";
+	let orcidTextStatusIsInProfileTrue  = "orcid.publication.inProfile.true";
+	let orcidTextStatusIsInProfileFalse = "orcid.publication.inProfile.false";
 
-var orcidTextPublishUpdate = "Im ORCID Profil aktualisieren";
-var orcidTextPublishCreate = "Ins ORCID Profil übertragen";
-var orcidTextPublishConfirm = "Die Publikation wurde in Ihr ORCID Profil übertragen!";
+	let orcidTextPublishUpdate = "orcid.publication.action.update";
+	let orcidTextPublishCreate = "orcid.publication.action.create ";
+	let orcidTextPublishConfirm = "orcid.publication.action.confirmation";
+	const keyPrefix = "orcid.publication*";
 
-jQuery(document).ready(function() {
-	jQuery('div.orcid-status').each(function() {
-		getORCIDPublicationStatus(this);
-	});
-	jQuery('div.orcid-publish').each(function() {
-		showORCIDPublishButton(this);
-	});
-});
+	let docReady=false;
+	let langReady=false;
 
-function getORCIDPublicationStatus(div) {
-	var id = jQuery(div).data('id');
-	jQuery.get(orcidStatusURL + id, function(status) {
-		console.log(status);
-		setORCIDPublicationStatus(div, status);
+	jQuery.ajax(translateURL + currentLang +"/" + keyPrefix).done(function(data){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['currentLang' is not defined.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=5519111)